packages/ckeditor5-html-support/src/generalhtmlsupport.ts

```diff
@@ -124,7 +124,7 @@ export default class GeneralHtmlSupport extends Plugin {
 	 * @param className The css class to remove.
 	 * @param selectable The selection or element to update.
 	 */
-	private removeModelHtmlClass( viewElementName: string, className: ArrayOrItem<string>, selectable: LimitedSelectable ) {
+	public removeModelHtmlClass( viewElementName: string, className: ArrayOrItem<string>, selectable: LimitedSelectable ): void {
```

Comment:
Make it internal. I know it's used outside this package, but it shouldn't be used by anyone (`ckeditor5-style` is an exception).